monkey/monkey_island/cc/ui/src/components/report-components/security/issues/AzurePasswordIssue.js

```diff
@@ -1,4 +1,5 @@
 import React from 'react';
```

Comment:
Shouldn't this entire commit be squashed into the original one?

Reply:
Why do you think that it should?